.aws/src/event-rules/account-delete-monitor/index.ts

```diff
@@ -0,0 +1,61 @@
+import { Construct } from 'constructs';
+import { Resource } from 'cdktf';
+import {
+  PocketEventBridgeProps,
+  PocketEventBridgeRuleWithMultipleTargets,
+  ApplicationEventBus,
+} from '@pocket-tools/terraform-modules';
+import { config } from '../../config';
+import { sns, sqs } from '@cdktf/provider-aws';
+import { config as admConfig } from './config';
+
+export class AccountDeleteMonitorEvents extends Resource {
+  public readonly snsTopic: sns.SnsTopic;
+  public readonly sqs: sqs.DataAwsSqsQueue;
+  public readonly sqsDlq: sqs.DataAwsSqsQueue;
+
+  constructor(
+    scope: Construct,
+    name: string,
+  ) {
+    super(scope, name);
+    // pre-existing queues (prod and dev) created by account-delete-monitor
+    this.sqs = new sqs.DataAwsSqsQueue(this, `${admConfig.prefix}-queue`, {
```

Comment:
hi just confirming -  we are publishing the `queue-check-delete` event to sqs instead of lambda? 

https://app.diagrams.net/#G1hwBjTaPYNLYXyjfB2laYG-cR4wN4Y-g1
![Screen Shot 2022-08-03 at 10 38 01 AM](https://user-images.githubusercontent.com/12415584/182673386-e173ddd1-69ab-46ad-ab1e-b9ab4cc60ac4.png)

Reply:
also, `The consuming function will handle any logic regarding how to handle this event.`

just making sure the lambda would send the `check-delete` event with userId payload back to the event-bridge (and not directly contact the service), so list/annotations/account deletion can consume them. is my understanding right? 

![Screen Shot 2022-08-03 at 10 42 29 AM](https://user-images.githubusercontent.com/12415584/182674391-d001383d-f19b-429d-a52c-79cd6e1e6a51.png)